Sample-01/src/app/app-routing.module.ts

```diff
@@ -24,7 +24,7 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [RouterModule.forRoot(routes, { relativeLinkResolution: 'legacy' })],
```

Comment:
I was reading about `relativeLinkResolution` here https://angular.io/api/router/ExtraOptions.
>Enables a bug fix that corrects relative link resolution in components with empty paths.

Does that mean we were not specifying the path correctly before?

Reply:
I do not think it does. This change has been added in automatically as a matter of upgrading. The legacy is the one that is backwards compatible with pre-11.

Let me get rid of this and use the default version, which is "corrected" and should work for us.

Reply:
Removed it 👍